wake-lock/wakelock-query.https.any.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+test(() => {
+  const invalidTypes = [
+    "invalid",
+    "",
+    null,
+    true,
+    123,
+    {},
+    Symbol(),
+    () => {},
+    self
+  ];
+  const lock = new WakeLock("screen");
```

Comment:
what about no argument?

Reply:
Uh, sorry, I didn't get the point. Isn't the "WakeLockType" a required argument for the constructor?

Reply:
but you want to check that it fails no? :)

Reply:
That'd been tested in this file: https://github.com/web-platform-tests/wpt/blob/master/wake-lock/wakelock-type.https.html

And here, I intended to test the invalid type argument for `query()` method.